packages/core/bootstrap/src/lib/modules/overrider.ts

```diff
@@ -52,23 +52,24 @@ export class Overrider {
     const isOverridden: { [symbol: string]: boolean } = {}
     const alreadyWarned: { [symbol: string]: boolean } = {}
     for (const coinResponse of coinsResponse) {
-      const symbol = coinResponse.symbol
+      const symbolUpperCase = coinResponse.symbol.toUpperCase()
+      const symbolLowerCase = coinResponse.symbol.toLowerCase()
 
-      if (remainingSyms.includes(symbol)) {
-        if (isOverridden[symbol] === true) {
-          if (!alreadyWarned[symbol])
-            logger.warn(
-              `Overrider: The symbol "${coinResponse.symbol}" has a duplicate coin id and no override.`,
+      if (remainingSyms.includes(symbolUpperCase) || remainingSyms.includes(symbolLowerCase)) {
```

Comment:
I'm worried about the run-time characteristics for this if the list of coins reaches into the hundreds. I'd recommend making `remainingSyms` into a map using the uppercase symbols, then turning the `includes` checks into `if (remainingSymsMap[symbolUpperCase])`.